scripts/generate_code.sh

```diff
@@ -7,9 +7,11 @@ if [ -z "$PT_INC_DIR" ]; then
   PT_INC_DIR="$PTDIR/build/aten/src/ATen"
 fi
 
+set -e
```

Comment:
The new `impl_path` argument is to the codegen can read in the kernel signatures and figure out if any are missing.

I also had to add `set -e` to ensure that the build actually stops if codegen fails, instead of just continuing. Since that would obscure the codegen error and you'd end up with the same linker error. I'm not sure if the original xla codegen suffered from that same issue though.